### PR TITLE
Update script

### PR DESCRIPTION
**Pull request by BrandonRomano**
May need to rebase back onto the `readme` branch, but this is good to go + ready to be hooked up to a cronjob

### CODE REVIEW

**Comment by kylemac** [comment]
so i'm thinking we create a `origin/bump` branch and we force push a timestamp. this way there's not 1M commits and the timestamp doesn't have to actually live on `origin/master`. it'll still serve the same purpose to keep the repo at the top of the list.

**Comment by BrandonRomano** [comment]
Good call

File: README.md [hunk 1]
```diff
@@ -30,3 +30,5 @@ If you're curious about the tools, stacks and technologies we use, get ready for
 - apply directly via [our hiring page](http://carrot.is/hiring)
 - [file an issue](https://github.com/carrot/we-are-hiring/issues) with a question you might have
 - fork this repository and PR in your resume/application (or just something we might find interesting)
+
+Last Updated: 04-10-2015 08:27 AM
```
Comment (notslang) [inline README.md]: don't we already know when it was last updated from the git commit? Like `git log -1 --format=%cd README.md`?

Comment (kylemac) [inline README.md]: : ) 

the script is going to make a throwaway commit on a throwaway branch every X hours (on a cron) to keep the repo relatively at the top of our http://github.com/carrot listing

Comment (notslang) [inline README.md]: oh, that makes more sense - maybe we could move the change to a separate file like `./.bump` so it doesn't pollute the commit history of the readme.

Comment (kylemac) [inline README.md]: we won't merge the `origin/bump` branch into master and it'll just `--amend` the commit each time. just the minimum to keep it at the top of the list

**Comment by BrandonRomano** [comment]
@kylemac If this is ready to merge, can you merge it in?  Or are we just going to keep the code out of the master branch?

**Pull request merged**